core/types/transaction_signing.go

```diff
@@ -249,6 +253,78 @@ func (s pragueSigner) Hash(tx *Transaction) common.Hash {
 		})
 }
 
+// isthmusSigner skips cancun because blob txs are not supported on OP
+type isthmusSigner struct{ londonSigner }
```

Comment:
Couldn't we instead wrap a `pragueSigner` and then forbid blob txs? Seems like this would need less code duplication.

Reply:
good idea - updated!